go.mod

```diff
@@ -0,0 +1,32 @@
+module github.com/profefe/profefe
```

Comment:
Adding go modules is a separate big topic by itself (see https://github.com/profefe/profefe/issues/38). I'd prefer to avoid mixing them with the changes in this PR.

Reply:
Ok, it was just the quickest way to get it compile. I've happy to remove it again.

Reply:
Oh, that's interesting. It should compile just fine if you clone the repo in the GOPATH. Everything that's needed is in vendor. Please let me know if something doesn't work for you,